spotseeker_server/cache/memory_cache.py

```diff
@@ -0,0 +1,100 @@
+""" Copyright 2012, 2013 UW Information Technology, University of Washington
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+"""
+from spotseeker_server.models import Spot
+from django.conf import settings
+from sys import maxint
+
+spots_cache = {}
+spot_cache_limit = getattr(settings, 'SPOTSEEKER_SPOT_CACHE_LIMIT', maxint)
+
+
+def get_spot(spot_model):
+    """Retrieves the cached version of the spot with the provided ID."""
+    verify_cache(spot_model)
```

Comment:
It looks like verify_cache is primarily built on methods that set values into the cache, so it shouldn't be the first thing called when someone's getting the cache